.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -9,31 +9,18 @@ labels: 'type:bug, status:requirements, priority-5-triage'
       Email renovate-disclosure@whitesourcesoftware.com instead.
 -->
 
-**What Renovate type, platform and version are you using?**
-
-<!-- Tell us if you're using the hosted App, or if you are self-hosting Renovate yourself. Platform too (GitHub, GitLab, etc) plus which version of Renovate if you're self-hosted. -->
-
 **Renovate type**
 
 - [ ] Hosted app
-- [ ] Self hosted
-
-**Renovate platform**
-
-- [ ] Azure DevOps (dev.azure.com)
-- [ ] Azure DevOps Server (self-hosted)
-- [ ] Bitbucket Cloud (bitbucket.org)
-- [ ] Bitbucket Server (self-hosted)
-- [ ] Gitea (gitea.com, self-hosted)
-- [ ] GitHub (github.com, GitHub Enterprise)
-- [ ] GitLab (gitlab.com, self-hosted)
+- [ ] Self hosted:
 
-**Renovate version (if self hosting)**
+Platform: ...
 
-I'm self hosting Renovate with version: ...
+Renovate version: ...
```

Comment:
So we also need to know the Renovate version if people are using the hosted app?

Reply:
No